classes/class-wc-connect-shipping-label.php

```diff
@@ -10,12 +10,13 @@ public function __construct( WC_Connect_Service_Settings_Store $settings_store )
 			$this->settings_store = $settings_store;
```

Comment:
Git messed up this file diff a bit. It's better if you look at the final code of `get_individual_packages()` and `get_packaging_data()`